src/App.js

```diff
@@ -1,18 +1,37 @@
 import React from 'react';
 import './App.css';
-import ViewSpace from './routes/view-space/the-think-tank'
+import Settings from './routes/settings';
+import SurveyView from './routes/survey';
+import ViewSpace from './routes/view-space/the-think-tank';
 import Homebar from './components/homebar';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+} from "react-router-dom";
 
-class App extends React.Component {
-  constructor (props){
-    super (props)
-  }
-  render () {
-    return ([
-      <ViewSpace />, // FIXME / TODO : TEMPORARILY FOR TESTING
+export default function App() {
+  return (
+    <Router>
```

Comment:
I think I recall a different way to do react routing that might be cleaner (?), but I cant remember off the top of my head. I'll have to look into it, but for now this is fine.